src/cryptenroll/cryptenroll-fido2.c

```diff
@@ -75,7 +77,8 @@ int enroll_fido2(
                                        JSON_BUILD_PAIR("keyslots", JSON_BUILD_ARRAY(JSON_BUILD_STRING(keyslot_as_string))),
                                        JSON_BUILD_PAIR("fido2-credential", JSON_BUILD_BASE64(cid, cid_size)),
                                        JSON_BUILD_PAIR("fido2-salt", JSON_BUILD_BASE64(salt, salt_size)),
-                                       JSON_BUILD_PAIR("fido2-rp", JSON_BUILD_STRING("io.systemd.cryptsetup"))));
+                                       JSON_BUILD_PAIR("fido2-rp", JSON_BUILD_STRING("io.systemd.cryptsetup")),
+                                       JSON_BUILD_PAIR("fido2-pin-required", JSON_BUILD_BOOLEAN(lock_with_pin))));
```

Comment:
hmm, fido2 spec calls this "clientPn". Maybe we should stick to that naming, and call this "fido2-clientPin".

Reply:
ok, but I'll leave the suffix `-required`, as it's not storing the actual pin, but just a flag